nautobot/dcim/templates/dcim/inc/cable_termination.html

```diff
@@ -20,6 +20,14 @@
                 <a href="{{ termination.get_absolute_url }}">{{ termination }}</a>
             </td>
         </tr>
+    {% elif termination.power_panel %}
+        {# Device Power Panel #}
+        <tr>
+            <td>Power Feed</td>
```

Comment:
I think that, similar to the other cases, we'd want to include both the power panel (`termination.power_panel`) and the power feed itself (`termination`) in the page.